src/de_main.c

```diff
@@ -404,7 +404,7 @@ static void Game_Iterate(AppState *app)
         Uint64 new_frame_time = SDL_GetTicks();
         Uint64 delta_time = new_frame_time - app->frame_time;
         app->frame_time = new_frame_time;
-        app->dt = delta_time * (0.001f);
+        app->dt += delta_time * (0.001f);
```

Comment:
Functionality LGTM but I would leave `app->dt` as is and create a new variable for this.
Physics will use fixed timestep but we might still want to have access to real dt for some things - like animating UI etc.